spec/system/admin/orders_spec.rb

```diff
@@ -179,6 +186,7 @@
       fill_in "Email", with: user.email
       fill_in "First name begins with", with: "J"
       fill_in "Last name begins with", with: "D"
+byebug
```

Comment:
You can remove this from the Git history before submitting a PR. The easiest way I find is to make a single commit that just removes the line, then do an interactive rebase and put the fixup commit directly under the commit with the mistake and mark it as "fixup". The two commits then get merged into one and it's like this line never happened. :smile: